modules/460-log-shipper/hooks/internal/vector/transform/extra_fields.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package transform
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+// Copied these regexes from another place. Remove them after refactoring.
```

Comment:
I saw these regexps somewhere else. Don't you want to remove them from here

Reply:
As the comment states, the idea is to delete these regex completely after refactoring.